INFO     root:log_util.py:16 接口用例数据：[3.0, '密码错误，登录成功', 'http://39.100.104.214:8080', '/login/', 'post', '{"username":"juno${unique_num2}","password":"juno999999"}', '"code": "03"', '', '', '']
INFO     root:log_util.py:16 请求原始数据：{"username":"juno${unique_num2}","password":"juno999999"}
INFO     root:log_util.py:16 请求数据预处理结果：{"username":"juno30302","password":"juno999999"}
INFO     root:log_util.py:16 接口调用成功！
INFO     root:log_util.py:16 响应数据：{"code": "03", "params": {"username": "juno30302", "password": "juno999999"}}
INFO     root:log_util.py:16 无关联参数提取..
INFO     root:log_util.py:16 【"code": "03"】关键字断言成功【{"code": "03", "params": {"username": "juno30302", "password": "juno999999"}}】